contracts/oracle/morpho/mainnet/MorphoFeedPTUSDe.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.12;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+import "../../BaseFeedPTPendle.sol";
+
+/// @title MorphoFeedPTUSDe
+/// @author Angle Labs, Inc.
+/// @notice Gives the price of PT-USDe in ETH in base 18
+contract MorphoFeedPTUSDe is BaseFeedPTPendle {
+    string public constant description = "PT-USDe/USDe Oracle";
```

Comment:
PT-USDe/USD oracle no?
Rather than PT-USDe/USDe oracle

Reply:
No this just allow to price the PT to USDe we then need an oracle USDe-->USD (Morpho factory)